blackbox/docs/admin/privileges.rst

```diff
@@ -56,11 +76,34 @@ and ``DELETE`` statements, on the object for which the privilege applies.
 .......
 
 Granting ``Data Definition Language (DDL)`` privilege to a user, indicates that
-this user is allowed to execute ``CREATE TABLE``, ``DROP TABLE``,
-``CREATE VIEW``, ``DROP VIEW``, ``CREATE FUNCTION``, ``DROP FUNCTION``,
-``CREATE REPOSITORY``, ``DROP REPOSITORY``, ``CREATE SNAPSHOT``,
-``DROP SNAPSHOT``, ``RESTORE SNAPSHOT``, and ``ALTER`` statements, on the
-object for which the privilege applies.
+this user is allowed to execute the following statements on objects for which
+the privilege applies:
+
+- ``CREATE TABLE``
+- ``DROP TABLE``
+- ``CREATE VIEW``
+- ``DROP VIEW``
+- ``CREATE FUNCTION``
+- ``DROP FUNCTION``
+- ``CREATE REPOSITORY``
+- ``DROP REPOSITORY``
+- ``CREATE SNAPSHOT``
+- ``DROP SNAPSHOT``
+- ``RESTORE SNAPSHOT``
+- ``ALTER TABLE``
+
+``AL``
+......
+
+Granting ``Administration Language (AL)`` privilege to a user, enables the user
+to execute the following statements:
+
+- ``CREATE USER``
+- ``DROP USER``
```

Comment:
Wasn't sure if we should also include it. It would open a way to gain additional privileges. (Currently you can only grant permissions you have, once you can change passwords you can take-over any other user who might already have more privileges..)

Reply:
Because one cannot set a pw for the `crate` superuser, I thought we do these features also to let users create their own superuser account. If this isn't the use-case, I agree to leave it out.

Reply:
I'd leave it out for now, it is always easier to add things later on instead of removing them again. (And it would imho invalidate the logic that users with AL can only grant privileges they also have)